### PR TITLE
unit tests for merging of two empty objects and two empty arrays

### PR DESCRIPTION
**Pull request by syfds**
Hi @hemantsonu20 , there were no unit test for empty values so I added two test files. Can I link this pull request somehow to the issue #4? 

What do you think about meaningful filenames in json-merge-test directory? So you can understand the test case without opening a file. 

Best regards,
Sergej

### CODE REVIEW

File: src/test/resources/json-merge-test/json-merge-test19.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[
+  {},
+  {},
+  {}
+]
```
Comment (hemantsonu20) [inline src/test/resources/json-merge-test/json-merge-test19.json:5]: this test case is already written in https://github.com/hemantsonu20/json-merge/blob/master/src/test/resources/json-merge-test/edge-cases/edge-case1.json

File: src/test/resources/json-merge-test/json-merge-test20.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[
+  [],
+  [],
+  []
+]
```
Comment (hemantsonu20) [inline src/test/resources/json-merge-test/json-merge-test20.json:5]: This is good, could you please move it to [json-merge-test/edge-cases](https://github.com/hemantsonu20/json-merge/blob/master/src/test/resources/json-merge-test/edge-cases/)

**Review by hemantsonu20 — CHANGES_REQUESTED** [review]

**Comment by hemantsonu20** [comment]
Thanks for the PR @syfds. 
I will link this PR to the issue #4.

Regarding `meaningful filenames for test files`
Initially I tried to rename file as per the test, but after few files I coudn't find suitable unique names as test cases were similar. It works well when you have multiple methods to test and you can rename test files according to method names.
If you have something in mind, open a new PR. Will surely consider it.
